src/app_service/service/settings/service.nim

```diff
@@ -985,3 +985,119 @@ QtObject:
     except ValueError:
       error "parse lastTokensUpdate: ", lastTokensUpdate
     return self.settings.lastTokensUpdate
+
+  ### News Feed Settings ###
+  proc notifSettingStatusNewsChanged*(self: Service) {.signal.}
+
+  proc toggleNewsFeedEnabled*(self: Service, value: bool): bool =
+    try:
+      let response = status_settings.toggleNewsFeedEnabled(value)
+      if not response.error.isNil:
+        raise newException(RpcException, response.error.message)
+
+      self.settings.newsFeedEnabled = value
+      self.notifSettingStatusNewsChanged()
+      return true
+    except Exception as e:
+      error "error: ", procName="toggleNewsFeedEnabled", errName = e.name, errDesription = e.msg
+      return false
+
+  proc getNotifSettingStatusNews*(self: Service): string {.slot.} =
+    result = VALUE_NOTIF_SEND_ALERTS # Default value
+
+    var newsFeedEnabled = false
+    var newsNotificationsEnabled = false 
+
+    if self.initialized:
+      newsFeedEnabled = self.settings.newsFeedEnabled
+      newsNotificationsEnabled = self.settings.newsNotificationsEnabled
+    else:
+      try:
+        var response = status_settings.newsFeedEnabled()
+        if not response.error.isNil:
+          error "error reading news feed enabled setting: ", errDescription = response.error.message
+          return
+        newsFeedEnabled = response.result.getBool
+
+        response = status_settings.newsNotificationsEnabled()
+        if not response.error.isNil:
+          error "error reading news notifications enabled setting: ", errDescription = response.error.message
+          return
+        newsNotificationsEnabled = response.result.getBool
+      except Exception as e:
+        let errDesription = e.msg
+        error "reading news settings error: ", errDesription
+        return
+
+    # We convert the bools to the right setting
+    # Send alerts means the News Feed is enabled + notifications are enabled
+    # Deliver quietly means the News Feed is enabled + notifications are disabled
```

Comment:
`+ notifications are disabled`?? Then this means that the UI should consider this "flag" to display the "Turn ON News Notification" panel on the AC tab??

I mean, if:
<img width="726" alt="Screenshot 2025-04-22 at 10 16 45" src="https://github.com/user-attachments/assets/0ea70e4b-a0ce-4ce9-9b1f-caf38761bcee" />

Then, this:
<img width="630" alt="Screenshot 2025-04-22 at 10 16 15" src="https://github.com/user-attachments/assets/4a21bfd2-a51e-4889-a56e-215f840dd4ba" />

Reply:
It's not done this way now. This panel is only shown when `Turn off` is set

Reply:
It's correct how it is. Maybe my comment is confusing or the settings choices. IMO, it would have been simpler if we had used an `enum` in status-go to match the combo box we have in the Settings, but it was decided to do two booleans. I guess it's to fit the mobile settings

So to summarize, there are two bool settings in status-go that map to that one setting in Desktop with three options.
The main setting is `newsFeedEnabled`. If that is disabled, there is no News Feed at all.
`newsNotificationsEnabled` is only about sending notifications (Push on mobile and OS/ephemeral on Desktop).

To make it clearer, I'll update the comment to say `OS notifications are disabled`

Reply:
So, what `Deliver Quietly` means? _newsFeedEnabled_ but _os notifications disabled_?

Reply:
> So, what `Deliver Quietly` means? _newsFeedEnabled_ but _os notifications disabled_?

Yes exactly. You'll receive the AC notifications, but no OS notifications. So you receive the News "quietly"